src/compaction_execute_bench.rs

```diff
@@ -232,8 +232,8 @@ fn run_bench(
     }
     info!("finished loading");
     let ssts: Vec<SSTableHandle> = sst_ids
-        .iter()
-        .map(|id| ssts_by_id.get(id).expect("expected sst").clone())
+        .into_iter()
```

Comment:
In this context, I believe it is more memory efficient to go with the later since it consumes the sst_ids vector and then uses it to produce a new vector in its place. This results in just one object being transformed in place . What we had before results in new two objects where the source object is actually useless and a memory waste once the sink object is produced.

Reply:
Yeah thats' right we dont need the variable anymore after the loop so we are just consuming it